docs/schema/v0/targeting.json

```diff
@@ -0,0 +1,581 @@
+                "pattern": "^\\$flagd\\..*$"
+              }
+            },
+            {
+              "type": "array",
+              "$comment": "this is to support the form of var with a default... there seems to be a bug here, where ajv gives a warning (not an error) because maxItems doesn't equal the number of entries in items, though this is valid in this case",
+              "minItems": 1,
+              "items": [
+                {
+                  "type": "string"
+                }
+              ],
+              "additionalItems": {
+                "anyOf": [
+                  {
+                    "type": "null"
+                  },
+                  {
+                    "type": "boolean"
+                  },
+                  {
+                    "type": "string"
+                  },
+                  {
+                    "type": "number"
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      }
+    },
+    "missingRule": {
+      "title": "Missing Operation",
+      "description": "Takes an array of data keys to search for (same format as var). Returns an array of any keys that are missing from the data object, or an empty array.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "missing": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "missingSomeRule": {
+      "title": "Missing-Some Operation",
+      "description": "Takes a minimum number of data keys that are required, and an array of keys to search for (same format as var or missing). Returns an empty array if the minimum is met, or an array of the missing keys otherwise.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "missing_some": {
+          "minItems": 2,
+          "maxItems": 2,
+          "type": "array",
+          "items": [
+            {
+              "type": "number"
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          ]
+        }
+      }
+    },
+    "binaryOrTernaryOp": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 3,
+      "items": {
+        "$ref": "#/$defs/args"
+      }
+    },
+    "binaryOrTernaryRule": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "if": {
+          "title": "If Operator",
+          "description": "The if statement takes 3 arguments: a condition (if), what to do if its true (then), and what to do if its false (else, optional). Note that the form accepting more than 3 arguments (if/else) is not supported in flagd; use nesting instead.",
```

Comment:
I think this description is slightly outdated since the `if` condition can take 2 or 3 arguments.

Reply:
I'll fix this upstream, but if it's alright with you I'd like to release this without the update, since this is just a description and we even say a few words late the last param is optional.

Reply:
https://github.com/open-feature/flagd-schemas/pull/126